docs/src/snap/reference/annotations.md

```diff
@@ -1,35 +1,169 @@
 # Annotations
 
 This page outlines the annotations that can be configured during cluster
-[bootstrap]. To do this, set the cluster-config/annotations parameter in
+[bootstrap]. To do this, set the `cluster-config/annotations` parameter in
 the bootstrap configuration.
```

Comment:
include an example? I think we should get in the habit of doing this at the top of the file, since that is likely the only hint people will need to be effective.

Reply:
Done